lib/class-wp-theme-json.php

```diff
@@ -337,6 +339,7 @@ public function __construct( $contexts = array() ) {
 			// Process settings subtree.
 			$this->process_key( 'settings', $context, self::SCHEMA );
 			if ( isset( $context['settings'] ) ) {
+				$this->process_key( 'border', $context['settings'], self::SCHEMA['settings'] );
```

Comment:
This is unrelated and should have been added in the PR that added the border as settings, but I'm adding here.